src/minuteman_lashup_vip_listener.erl

```diff
@@ -192,6 +198,28 @@ code_change(_OldVsn, State, _Extra) ->
 %%% Internal functions
 %%%===================================================================
 
+handle_lookup_vips(State, Names) ->
+  lists:map(fun(Name) -> handle_lookup_vip(State, Name) end, Names).
+
+handle_lookup_vip(State, {ip, IP}) ->
```

Comment:
isn't that like creating shared memory?  what's the point of wrapping the other one as a fixture?